frontend/src/workbench/RightSite.tsx

```diff
@@ -11,93 +11,79 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
-import { makeStyles } from '@material-ui/core/styles';
-import Tab from '@material-ui/core/Tab';
-import Tabs from '@material-ui/core/Tabs';
+import MuiAccordion from '@material-ui/core/Accordion';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import MuiAccordionSummary from '@material-ui/core/AccordionSummary';
+import MuiCollapse from '@material-ui/core/Collapse';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
 import { PropertiesWebSocketContainer } from 'properties/PropertiesWebSocketContainer';
-import React, { useState } from 'react';
-import { ValidationWebSocketContainer } from 'validation/ValidationWebSocketContainer';
-import { RightSiteProps, TabPanelProps } from './RightSite.types';
+import React from 'react';
+import { RightSiteProps } from './RightSite.types';
 
 const useSiteStyles = makeStyles((theme) => ({
   site: {
-    display: 'flex',
-    flexDirection: 'column',
-    '& > *:nth-child(2)': {
-      flexGrow: 1,
-    },
-  },
-  tabsRoot: {
-    minHeight: theme.spacing(4),
-    borderBottomColor: theme.palette.divider,
-    borderBottomWidth: '1px',
-    borderBottomStyle: 'solid',
+    display: 'grid',
+    gridTemplateColumns: 'minmax(0,1fr)',
+    gridTemplateRows: 'minmax(0,1fr)',
   },
-  tabRoot: {
-    minHeight: theme.spacing(4),
-    textTransform: 'none',
+  accordionDetailsRoot: {
+    display: 'block',
   },
-  tabLabel: {
-    display: 'flex',
-    flexDirection: 'row',
-    alignItems: 'center',
-    '& > *:nth-child(2)': {
-      marginLeft: theme.spacing(1),
+}));
+
+const Accordion = withStyles({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: 'minmax(0,1fr)',
+    gridTemplateRows: 'min-content minmax(0,1fr)',
+    border: '1px solid rgba(0, 0, 0, .125)',
+    boxShadow: 'none',
+    '&:not(:last-child)': {
+      borderBottom: '0px',
+    },
+    '&:before': {
+      display: 'none',
+    },
+    '&$expanded': {
+      margin: '0px',
     },
   },
-}));
+  expanded: {},
+})(MuiAccordion);
 
-const a11yProps = (id: string) => {
-  return {
-    id: `simple-tab-${id}`,
-    'aria-controls': `simple-tabpanel-${id}`,
-  };
-};
+const AccordionSummary = withStyles({
+  root: {
+    borderBottom: '1px solid rgba(0, 0, 0, .125)',
+    marginBottom: -1,
+    '&$expanded': {
+      minHeight: 48,
+    },
+  },
+  content: {
+    '&$expanded': {
+      margin: '0px',
+    },
+  },
+  expanded: {},
+})(MuiAccordionSummary);
 
-const TabPanel = ({ children, value, index }: TabPanelProps) => {
-  return (
-    <div
-      role="tabpanel"
-      hidden={value !== index}
-      id={`simple-tabpanel-${index}`}
-      aria-labelledby={`simple-tab-${index}`}>
-      {value === index && <div>{children}</div>}
-    </div>
-  );
-};
+const customCollapse = withStyles({
```

Comment:
Use an upper case letter here too `CustomCollapse` since it's a component

Reply:
Done